Automattic/woocommerce-payments
# Use WC Admin package for merchant onboarding tests, fix merchant E2E tests

Fixes #3482 

#### Changes proposed in this Pull Request
- Move merchant onboarding tests to use `@woocommerce/admin-e2e-tests`, refactor accordingly.
- Fix failing merchant admin E2E tests.

#### Testing instructions
- Refactored tests so try running the updated tests individually, for example:
```
npm run test:e2e-dev tests/e2e/specs/merchant/merchant-orders-manual-capture.spec.js      
```
- Updated tests should pass on this branch and fail on `develop`.

<img width="842" alt="Screenshot 2021-12-13 at 3 38 04 PM" src="https://user-images.githubusercontent.com/68396823/145792667-ce916bc1-9380-4905-b551-acbbcc4654c7.png">


-------------------


**Post merge**
